src/components/app.js

```diff
@@ -2,28 +2,45 @@ import React from 'react';
 import Search from './search/search.js';
 import Gender from './gender/gender.js';
 import Distance from './distance/distance.js';
-import Results from './results/results.js';
+import DoctorInfoList from './results/doctorInfoList.js';
+import TotalResults from './results/totalResults.js';
 import data from '../api/doctors.json'
 
-const App = () => {
-    return (
-        <>
-           <Search />
-           <div className="content content--2col-left">
-				<div className="results-container is-vishiden">
-					<div className="sidebar-content">
-						<form className="js-filters-form">
-                            <Distance />
-                            <Gender />
-						</form>
-					</div>
-					<div className="general-content js-general-content ">
-						<Results resultsData={data}/>
+class App extends React.Component {
+    constructor() {
+		super();
+		this.state = {      
+			results:  []  
+		};  
+	}
+
+	componentDidMount() {
+		this.setState({
+			results: data.results 
+		});
+	}
+	  
+	render() {
+		return (
+			<>
+			   <Search />
+			   <div className="content content--2col-left">
+					<div className="results-container is-vishiden">
```

Comment:
[question/fix] Why does your sidebar column that contains filters have a class of `results-container`? Also, why is it hidden?